garnet.py

```diff
@@ -25,25 +25,25 @@
 # set the debug mode to false to speed up construction
 set_debug_mode(False)
 
-
```

Comment:
you can do `import magma as m` and then `m.` instead of `magma.`. just more convenient